src/test/scala/deduplication/DeduplicationSuite.scala

```diff
@@ -119,4 +118,129 @@ class DeduplicationSuite extends FunSuite {
       }
   }
 
+  test(
```

Comment:
missing started, completed but expired is None? @filosganga

Reply:
good point

Reply:
It is the first case, isn't it?